dingo/MetabolicNetwork.py

```diff
@@ -108,30 +115,51 @@ def fva(self):
         """A member function to apply the FVA method on the metabolic network."""
 
         if self._parameters["fast_computations"]:
-            return fast_fva(
+            min_fluxes, max_fluxes, max_biomass_flux_vector, max_biomass_objective = fast_fva(
                 self._lb,
                 self._ub,
                 self._S,
                 self._biomass_function,
                 self._parameters["opt_percentage"],
             )
         else:
-            return slow_fva(
+            min_fluxes, max_fluxes, max_biomass_flux_vector, max_biomass_objective = slow_fva(
                 self._lb,
                 self._ub,
                 self._S,
                 self._biomass_function,
                 self._parameters["opt_percentage"],
             )
+        self._min_fluxes = min_fluxes
```

Comment:
My initial thought was about the [`sample_from_fva_output`](https://github.com/GeomScale/dingo/blob/c51f0a5b6a461e9405a548f1ae44f35d4bee86bf/dingo/PolytopeSampler.py#L293) function that used to get as arguments the `min_fluxes` and the `max_fluxes`. 

At first, I thought of keeping those in the model to provide them in the function but then I thought it would be simpler if the fva is performed with the `sample_from_fva_output` function. 

Yet, maybe it's practical to keep them. No strong arguments on that.

Reply:
Sorry, I was lost a bit. Are those class variables used even after that changes of this PR? As far as understand [sample_from_fva_output](https://github.com/GeomScale/dingo/blob/c51f0a5b6a461e9405a548f1ae44f35d4bee86bf/dingo/PolytopeSampler.py#L293) function is not using them.